applications/dashboard/controllers/api/AuthenticateApiController.php

```diff
@@ -255,7 +255,7 @@ public function post(array $body) {
             ],
             'persist:b' => [
                 'default' => false,
-                'description' => 'Set the persist option on the cookie when it is set.'
+                'description' => 'Set the persist option on the cookie when it is set.',
```

Comment:
![](https://media.giphy.com/media/jchcqpyTpIki4/giphy.gif)

Reply:
You better be. It's gaining support: https://wiki.php.net/rfc/trailing-comma-function-calls